docs/products/opensearch/howto/custom-repositories.md

```diff
@@ -35,6 +35,11 @@ Configure custom repositories in Aiven for OpenSearch to store [snapshots](/docs
   and
   [snapshot repository permissions](https://docs.opensearch.org/docs/latest/security/access-control/permissions/#snapshot-repository-permissions)
   configured
+- For calling the native OpenSearch
```

Comment:
```suggestion
- To use the native OpenSearch
```
How about using "To use" instead of "Calling". I felt it reads a bit more clearly.